hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/AbstractScheduleHandler.java

```diff
@@ -60,12 +60,21 @@ abstract class AbstractScheduleHandler {
 
     /**
      * A simple record to return both "deemed valid" signatories and remaining primitive keys that must sign.
-     * @param updatedSignatories a Set of "deemed valid" signatories, possibly updated with new entries
+     *
+     * @param updatedSignatories    a Set of "deemed valid" signatories, possibly updated with new entries
      * @param remainingRequiredKeys A Set of Key entries that have not yet signed the scheduled transaction, but
-     *     must sign that transaction before it can be executed.
+     *                              must sign that transaction before it can be executed.
      */
     protected static record ScheduleKeysResult(Set<Key> updatedSignatories, Set<Key> remainingRequiredKeys) {}
 
+    /**
+     * Gets the set of all the keys required to sign a transaction.
+     *
+     * @param scheduleInState the schedule in state
+     * @param context    the Prehandle context
+     * @return the set of keys required to sign the transaction
+     * @throws PreCheckException the pre check exception
```

Comment:
I'd prefer a little more for the `@throws`, or to not have it at all. Seems like it's just clutter the way it's written

Reply:
Thanks @mhess-swl. Updated.